src/PrestaShopBundle/ApiPlatform/Denormalizers/DecimalNumberDenormalizer.php

```diff
@@ -26,17 +26,20 @@
 
 declare(strict_types=1);
 
-namespace PrestaShopBundle\ApiPlatform\Converters;
+namespace PrestaShopBundle\ApiPlatform\Denormalizers;
 
-class StringToIntConverter implements ConverterInterface
+use PrestaShop\Decimal\DecimalNumber;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+class DecimalNumberDenormalizer implements DenormalizerInterface
 {
-    public function convert(mixed $value): int
+    public function denormalize($data, string $type, string $format = null, array $context = [])
```

Comment:
Also I wonder if we should use `Normalizer` as the namespace here, like Symfony does Because normalization and renormalization are part of the same concept and if we start introducing our custom denormalizers there are a lot of chances that at some point we'd need the opposite action (normalize) so the `Normalizer` namespace would be more generic and adapted maybe